thymleafdemo/src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+#
+# JDBC properties
+#
+spring.datasource.url=jdbc:mysql://localhost:3306/employee_directory
+spring.datasource.username=springstudent
+spring.datasource.password=springstudent
```

Comment:
@avantibonde where are the db migratoins? If haven't planned to push yet, I will suggest to use migration tool such as flyway or liquibase.
Flyway is easy to setup